testcases/grub.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+# OpenPOWER Automated Test Project
+#
+# Contributors Listed Below - COPYRIGHT 2017
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+
+import unittest
+import time
+import pexpect
+
+import OpTestConfiguration
+import OpTestLogger
+from common.OpTestSystem import OpSystemState
+from common.OpTestError import OpTestError
+
+log = OpTestLogger.optest_logger_glob.get_logger(__name__)
+
+
+class Grub(unittest.TestCase):
+    '''
+    Test case to measure boot timing
+    '''
+    def setUp(self):
+        self.conf = OpTestConfiguration.conf
+        self.cv_SYSTEM = self.conf.system()
+        self.cv_HOST = self.conf.host()
+        self.console = self.cv_SYSTEM.console
+
+    def measure_boot_time(self):
```

Comment:
Can we pick another name to convey the exact use case? boot_time is very generic.